com.github.SubhadeepJasu.pebbles.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
```

Comment:
You should use the desktop/appdata files from your upstream repository, there's no need to duplicate it here.